composer.json

```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
```

Comment:
that's strange to upgrade php version on 1.12 branch?

Reply:
https://github.com/Sylius/Sylius-Standard/pull/932#issuecomment-1724887847

Sylius-Standard doesn't follow SemVer (or at least it doesn't have to).

Reply:
Anyway, our new teammate @Wojdylak will check what's wrong with 8.0 :p. So we'll wait with merging for results from him investigation.

Reply:
> [#932 (comment)](https://github.com/Sylius/Sylius-Standard/pull/932#issuecomment-1724887847)
> 
> Sylius-Standard doesn't follow SemVer (or at least it doesn't have to).

Yeah indeed, I thought it was on sylius/sylius, misread

Reply:
In php 8.0, the sylius/sylius package installs in version 1.12.9, and there is an issue with doctrine/orm

Reply:
Could you add a conflict / write a version of the doctrine package?

Reply:
#933 I'm on it